libcudacxx/test/libcudacxx/CMakeLists.txt

```diff
@@ -126,6 +126,7 @@ if (NOT LIBCUDACXX_TEST_WITH_NVRTC)
   # Build but don't run the tests. Used by CI to pre-seed sccache for the test machines.
   # Only executed if explicitly requested.
   add_custom_target(libcudacxx.test.lit.precompile
+    DEPENDS libcudacxx.test.public_headers libcudacxx.test.internal_headers
```

Comment:
Why? Because I find it annoying when lit runs during the build of the other units. Plus it's probably doubling up on parallel build counts.